index.html

```diff
@@ -25,61 +27,111 @@
         </h3>
         <h3 class="text-primary font-mulish-bold text-[24px]">Trafalgar</h3>
       </div>
-      <nav>
-        <img src="/assets/images/List.svg" class="cursor-pointer" />
+      <nav class="flex lg:justify-center lg:gap-[40px]">
+        <a
+          href=""
+          class="text-dark-blue font-mulish-medium hidden text-[18px] lg:block"
+          >Home</a
+        >
+        <a
+          href=""
+          class="text-secondary font-mulish-medium hidden text-[18px] transition-all duration-150 hover:text-black lg:block"
+          >Find a doctor</a
+        >
+        <a
+          href=""
+          class="text-secondary font-mulish-medium hidden text-[18px] transition-all duration-150 hover:text-black lg:block"
+          >Apps</a
+        >
+        <a
+          href=""
+          class="text-secondary font-mulish-medium hidden text-[18px] transition-all duration-150 hover:text-black lg:block"
+          >Testimonials</a
+        >
+        <a
+          href=""
+          class="text-secondary font-mulish-medium hidden text-[18px] transition-all duration-150 hover:text-black lg:block"
+          >About us</a
+        >
+        <a
+          href=""
+          class="text-secondary font-mulish-medium hidden text-[18px] transition-all duration-150 hover:text-black lg:block"
+          >Login</a
+        >
+        <img src="/assets/images/List.svg" class="cursor-pointer lg:hidden" />
       </nav>
     </header>
-    <main class="flex flex-col items-center gap-[84px]">
-      <section class="flex flex-col items-start gap-[38px]">
-        <div class="flex max-w-[307px] flex-col items-start gap-[25px]">
+    <main class="flex flex-col items-center gap-[84px] lg:gap-[190px]">
+      <section
+        class="flex flex-col items-start gap-[38px] lg:w-full lg:flex-row lg:items-center lg:justify-center lg:gap-[109px] lg:px-[5px] xl:px-[190px]"
```

Comment:
suggestion: for future cases we prefer to use Tailwind's built-in spacing scale for consistency and responsiveness, instead of hardcoded pixel values